ui/component/claimListDiscover/index.js

```diff
@@ -13,6 +13,7 @@ import { selectFollowedTags } from 'redux/selectors/tags';
 import { selectMutedAndBlockedChannelIds } from 'redux/selectors/blocked';
```

Comment:
In Channel Home, if I expand "Show more upcoming", it overlaps with the Home's featured.  

<img width="660" alt="image" src="https://github.com/OdyseeTeam/odysee-frontend/assets/64950861/531d4427-dc9c-45d4-94d4-a46b1de28f8f">